lookout/style/format/code_generator.py

```diff
@@ -36,7 +38,7 @@ def __init__(self,
                  skip_errors: bool = False,
                  verbose: bool = False,
                  url: str = "<N/A>",
-                 commit: str = "<N/A>"):
+                 commit: str = "<N/A>") -> None:
```

Comment:
I am actually against these weird `-> None` type annotations for `__init__`. I do understand that they are required for mypy, but we are not going to use it, and it is basically the problem with mypy, not ours.